### PR TITLE
Match appveyor requirements install to travis

### PR DESCRIPTION
**Pull request by mtreinish**
### Summary

In #2328 tests were added that required nxpd to be installed, but it was
merged before waiting for the results from appveyor. These new tests
don't pass appveyor because the requirements installed were different
from what were installed on travis. This commit fixes this so that we
install the same set of requirements on appveyor and travis jobs to
make sure when people take shortcuts in the future there are less
surprises.

### Details and comments

### CODE REVIEW

File: test/python/visualization/test_circuit_matplotlib_drawer.py [hunk 1]
```diff
@@ -35,6 +36,7 @@ def _expected_empty(self):
 
     @unittest.skipIf(not visualization.HAS_MATPLOTLIB,
                      'matplotlib not available.')
+    @unittest.skipIf(os.name == 'nt', 'tempfile fails on appveyor')
```
Comment (1ucian0) [inline test/python/visualization/test_circuit_matplotlib_drawer.py]: I remember a conversation about temporary files in Windows... Cannot be solved by using `TemporaryFile`? 

Is this related with the depency issue?  Or a story for another PR?

Comment (mtreinish) [reply test/python/visualization/test_circuit_matplotlib_drawer.py]: I just copied this skip from another test, there might be a way in appveyor to get tempfiles to work it's a permissions issue in the way they configure the windows environments. But I think we should fix that in another pr, just to get things green again quickly

Comment (1ucian0) [reply test/python/visualization/test_circuit_matplotlib_drawer.py]: makes sense. I submitted https://github.com/Qiskit/qiskit-terra/issues/2335 to track it.
